cmd/odf/dr/dr.go

```diff
@@ -19,4 +21,6 @@ func init() {
 	// Add a subset of ramenctl commands suitable for "odf dr".
 	DrCmd.AddCommand(commands.InitCmd)
 	DrCmd.AddCommand(commands.TestCmd)
+
+	devpreview.Configure(DrCmd)
```

Comment:
Do the subcommands also need to call `Configure()` to get the dev preview description? Or does the dev preview description show for all subcommands already?

Reply:
It modifies the command and its children:
https://github.com/red-hat-storage/odf-cli/pull/175/commits/c0b4cc9e2726d4f5c1d312001d5cbb2e3fbdb03b#diff-b4456f00c100d04bae9cd3a1eed6c9c9e5a49b5b84c85535a5c9d34f991725d6R25